numba/_helperlib.c

```diff
@@ -577,8 +577,13 @@ import_cython_function(const char *module_name, const char *function_name)
         return NULL;
     cobj = PyMapping_GetItemString(capi, function_name);
     Py_DECREF(capi);
-    if (cobj == NULL)
+    if (cobj == NULL) {
+	PyErr_Clear();
+	PyErr_Format(PyExc_ValueError,
+		     "No function '%s' found in __pyx_capi__ of the module",
```

Comment:
Perhaps add the module name in here? Saves having to look through the traceback to find which module the function came from?